build/Dockerfile-go1.14.1-solc0.4.24-alpine-1.1

```
@@ -26,3 +26,4 @@ COPY --from=go_builder /usr/local/go /usr/local
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
 RUN mkdir -p "$GOPATH/src" "$GOPATH/bin" && chmod -R 777 "$GOPATH"
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.24.0
```

```suggestion
RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin
```
How about installing the latest version?

---

I'm not sure the latest version is always good. If we follow the latest, there is possibility that we encounter unexpected errors.